handbook/ops/finance/payables.md

```diff
@@ -0,0 +1,28 @@
+# Paying bills
+
+## Limits
+
+Generally, you should feel free to spend in ways that are in the best interest of the business (see our philosophy in [spending company money](../../people-ops/spending-company-money.md)). 
+
+However, if you intend to make a purchase above the limits below, please seek approval from your direct manager or, if necessary, [contact the finance team](index.md#contact)
+
+|Role|Maximum expense without management approval|Maximum expense without finance approval|
+|---|---|---|
+|Department leads (VP-level or above)|$15,000|$20,000|
```

Comment:
Just my two cents that these numbers are high, but not my call! At the very least I think finance can provide a good sounding board on whether the money is being well spent, if there are other options, etc...and from my experience there's no downside to have Ops in the know of when money is being spent, before it's spent.

Reply:
Haha, I hear you—but I think our culture needs to evolve along with our cash position and team. 15k isn't that much for an ad campaign or a piece of software. I'll get some more eyes on this once it's merged.